javascript/javascript2/week1/classwork/Exercise1/index.js

```diff
@@ -0,0 +1,75 @@
+function alertOnload() {
+    alert('hello, from js on load');
+}
+
+//alertOnload();
+
+const rootDiv = document.querySelector('.root-div')
+
+const button = document.createElement('button');
+button.innerText = 'Click me';
+rootDiv.appendChild(button);
+
+// Add an event listener for the button's click event.
+button.addEventListener("click", function () {
+    const para = document.createElement('p')
+    para.innerText = "you clicked me!"
+    rootDiv.appendChild(para);
+})
+
+
+// Logic for triggering and alert onclick.
+function alertonClick() {
+    alert('hello, from js click!');
+}
+
+//Try entering something into the field below, click somewhere else to trigger the event
+const input = document.querySelector('input');
```

Comment:
avoid variable with 'reserve keys' like input is a reserve key in JS

Reply:
sure Naresh.. It is corrected now. Thanks